Library/Homebrew/keg.rb

```diff
@@ -467,13 +466,13 @@ def oldname_opt_record
     end
   end
 
-  # TODO: refactor to use keyword arguments.
-  def link(**options)
+  def link(verbose: false, dry_run: false, overwrite: false)
     raise AlreadyLinkedError, self if linked_keg_record.directory?
 
     ObserverPathnameExtension.reset_counts!
+    options = { verbose: verbose, dry_run: dry_run, overwrite: overwrite }
```

Comment:
Does `optlink` support `dry_run`?

Reply:
`make_relative_symlink` in `optlink` refers `dry_run` like  https://github.com/Homebrew/brew/blob/e9a061768ffc002757bab495ecac5b1eb355f784/Library/Homebrew/keg.rb#L621